<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddPreset</class>
 <widget class="QDialog" name="DialogAddPreset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add font preset</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_currentPresetName">
     <property name="text">
      <string>Current font preset name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_baseFontPresetName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_newPresetName">
     <property name="text">
      <string>New font preset name</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_newFontPresetName</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_newFontPresetName">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_info">
     <property name="text">
      <string>Information</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
